README.md

```diff
@@ -6,76 +6,215 @@
     </picture>
 </h1>
 
-Welcome to the public home of Dependabot :dependabot:. This repository serves 2 purposes:
+Welcome to the public home of Dependabot :dependabot:.
 
-1. It houses the source code for Dependabot Core, which is the heart of [Dependabot][dependabot]. Dependabot Core handles the logic for updating dependencies on GitHub (including GitHub Enterprise), GitLab, and Azure DevOps. If you want to host your own automated dependency update bot then this repo should give you the tools you need. A reference implementation is available [here][dependabot-script].
-2. It is the public issue tracker for issues related to Dependabot's updating logic. For issues about Dependabot the service, please contact [GitHub support][support]. While the distinction between Dependabot Core and the service can be fuzzy, a good rule of thumb is if your issue is with the _diff_ that Dependabot created, it belongs here and for most other things the GitHub support team is best equipped to help you.
+# Table of Contents
 
-## Got feedback?
+- What is Dependabot-Core?
+- How to run Dependabot
+- Contributing to Dependabot
+  - Reporting Issues and Feature Requests
+  - Submitting Pull Requests
+- Development Guide
+  - Running the development environment
+  - Debugging problems
+  - Running Tests
+  - Profiling
+- Architecture Notes
+- License and Project History
+- Notes for Project Maintainers
```

Comment:
I think this is a reasonable TOC? 

I purposefully omitted some of the subheadings, in order to focus on the main entrypoints that folks will be looking for when they first hit the readme...

Reply:
I like it!